From 1619ea3f108e33384dcd33a57f1b1e548f90ecfb Mon Sep 17 00:00:00 2001
From: Alberto Mardegan <mardy@users.sourceforge.net>
Date: Mon, 6 Nov 2023 12:57:03 +0300
Subject: [PATCH 4/8] ag-provider: fix memory leak on provider tags

Commit a689b87e5a9185d36669822646f5f5bc57d8f846 adder support for tags
in provider files, but was not properly freeing the allocated memory.
---
 libaccounts-glib/ag-provider.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libaccounts-glib/ag-provider.c b/libaccounts-glib/ag-provider.c
index 8a79616..bd11ac9 100644
--- a/libaccounts-glib/ag-provider.c
+++ b/libaccounts-glib/ag-provider.c
@@ -545,6 +545,7 @@ ag_provider_unref (AgProvider *provider)
         g_clear_pointer (&provider->plugin_name, g_free);
         g_clear_pointer (&provider->file_data, g_free);
         g_clear_pointer (&provider->default_settings, g_hash_table_unref);
+        g_clear_pointer (&provider->tags, g_hash_table_unref);
         g_slice_free (AgProvider, provider);
     }
 }
-- 
2.44.0

